src/main/java/WayofTime/bloodmagic/ritual/IMasterRitualStone.java

```diff
@@ -22,6 +24,8 @@
 
     SoulNetwork getOwnerNetwork();
 
+    Map<String, AreaDescriptor> modableRangeMap = new HashMap<>();
```

Comment:
You're able to declare member fields in interfaces..?

Reply:
In fact I need to or my IDE complains.

Reply:
because Map itself is an interface, I guess...